benchmarks/express/benchmark.js

```diff
@@ -0,0 +1,78 @@
+const { spawn } = require("child_process");
+const { setTimeout } = require("timers/promises");
+const { promisify } = require("util");
+const exec = promisify(require("child_process").exec);
+
+// Accepted percentage of performance decrease
+const AcceptedDecrease = 50; // %
```

Comment:
```suggestion
const AcceptedDecrease = 50; // %
```

Increase maybe?

Reply:
Yes, thought it would be worse in GitHub Actions than locally.